examples/05_video_and_time.html

```diff
@@ -40,9 +40,9 @@
 					new TWEEN.Tween( { x: 50, y: 0 } )
 						.to( { x: 400 }, this.duration )
```

Comment:
I'm getting `NaN` for `this.duration` when testing this example locally. The video metadata might not be loaded yet when the 'play' event is triggered. Maybe we should use 'playing' event instead.

Reply:
I don't have this problem. Could you create a separate issue for this? I would like to get this fix merged.